### PR TITLE
revert steth

### PR DESCRIPTION
**Pull request by marktoda**
- Revert "add (un)/wrapping for steth (#317)"
- Revert "deploy UniversalRouter v1.3 to mainnet and goerli (#320)"
- fix: fixup steth leftovers

### CODE REVIEW

**Comment by marktoda** [comment]
https://github.com/Uniswap/universal-router-sdk/pull/153

File: README.md [hunk 1]
```diff
@@ -116,21 +116,7 @@ Each command is a `bytes1` containing the following 8 bits:
...
-   │ 0x1e │  ELEMENT_MARKET               │
-   ├──────┼───────────────────────────────┤
-   │ 0x1f │  -------                      │
-   ├──────┼───────────────────────────────┤
-   │ 0x20 │  SEAPORT_V1_4                 │
-   ├──────┼───────────────────────────────┤
-   │ 0x21 │  EXECUTE_SUB_PLAN             │
-   ├──────┼───────────────────────────────┤
-   │ 0x22 │  APPROVE_ERC20                │
-   ├──────┼───────────────────────────────┤
```
Comment (zhongeric) [inline README.md:128]: Don't think these rows should be deleted

File: README.md [hunk 2]
```diff
@@ -251,7 +237,7 @@ forge script --broadcast \
 --rpc-url <RPC-URL> \
 --private-key <PRIVATE-KEY> \
 --sig 'run()' \
-script/deployParameters/Deploy<network>.s.sol:Deploy<network> \
+script/deployParameters/Deploy<network>.s.sol:Deploy<network>
```
Comment (zhongeric) [inline README.md:240]: think we need the \

File: contracts/base/Dispatcher.sol [hunk 3]
```diff
@@ -351,24 +351,8 @@ abstract contract Dispatcher is NFTImmutables, Payments, V2SwapRouter, V3SwapRou
...
-                address recipient;
-                uint256 amountMin;
-                assembly {
-                    recipient := calldataload(inputs.offset)
-                    amountMin := calldataload(add(inputs.offset, 0x20))
-                }
-                Payments.unwrapSTETH(map(recipient), amountMin);
             } else {
-                // placeholder area for commands 0x25-0x3f
+                // placeholder area for commands 0x22-0x3f
```
Comment (zhongeric) [inline contracts/base/Dispatcher.sol:355]: should be 0x23

**Review by zhongeric — APPROVED** [review]
lgtm just some nits

File: README.md [hunk 4]
```diff
@@ -234,7 +230,7 @@ yarn test:gas
 
 ### To Deploy
 
-Fill out parameters in `script/deployParameters/Deploy<network>.s.sol`
+Fill out parameters in `script/deployParameters/Deploy<network>.s.sol` \
```
Comment (hensha256) [inline README.md:233]: Whats this for?

File: README.md [hunk 5]
```diff
@@ -126,11 +126,7 @@ Each command is a `bytes1` containing the following 8 bits:
    ├──────┼───────────────────────────────┤
    │ 0x22 │  APPROVE_ERC20                │
    ├──────┼───────────────────────────────┤
-   │ 0x23 │  WRAP_STETH                   │
-   ├──────┼───────────────────────────────┤
-   │ 0x24 │  UNWRAP_STETH                 │
-   ├──────┼───────────────────────────────┤
-   │ 0x25-│  -------                      │
+   │ 0x1e-│  -------                      │
```
Comment (hensha256) [inline README.md:129]: why 1e? Shouldnt it be 0x23?

File: README.md (hunk 2 above)
Comment (hensha256) [reply README.md:250]: agree with eric?

**Review by hensha256 — APPROVED** [review]
lgtm but gonna ask eric to re-review too

**Review by zhongeric — APPROVED** [review]

**Comment by lilyanB** [comment]
Hello, is it possible to know why you decided to stop supporting wrapping stETh/ unwrapping wstETH ? Thanks

**Pull request merged**